### PR TITLE
Add new Android features

### PR DESCRIPTION
**Pull request by dshokouhi**
https://github.com/home-assistant/home-assistant-android/pull/356 added several new notification options for Android users. 

- tags
- color
- sticky
- clickAction
- action URI

https://github.com/home-assistant/home-assistant-android/pull/336 added a couple new sensors.

- battery level
- wifi connection

### CODE REVIEW

**Comment by dshokouhi** [comment]
Should we be updating these docs alongside play store releases or whenever a commit is merged?  Users are free to install the firebase builds and those get generated on each commit.

**Comment by TomBrien** [comment]
I think just get the info out there. From iOS I think we saw people will run the earliest beta possible

**Comment by klaasnicolaas** [comment]
At least the new documentation on the website must be present at a release. 

I was personally very happy when I read the new features in the latest release, then I just didn't know where to get the documentation from and I couldn't find it because everything was described for iOS not for Android. (From the vision of a starting user)

We don't want users to walk away because they can't find their information.

File: docs/notifications/basic.md [hunk 1]
```diff
@@ -158,3 +190,60 @@ automation:
...
+        title: "Motion Detected in Backyard"
+        message: "Someone might be in the backyard."
+        data:
+          color: '#2DF56D' # or 'red'
+```
+
+### Sticky notification
+
+![android](assets/android.svg)
+You can set whether to dismiss the notification upon selecting it or not.  Setting `sticky` to `'true'` will keep the notification from being dismissed when the user selects it.  Setting it to `'false'` (default) will dismiss the notification upon selecting it.
```
Comment (klaasnicolaas) [inline docs/notifications/basic.md]:
```suggestion
You can set whether to dismiss the notification upon selecting it or not. Setting `sticky` to `'true'` will keep the notification from being dismissed when the user selects it. Setting it to `'false'` (default) will dismiss the notification upon selecting it.
```

File: docs/notifications/basic.md [hunk 2]
```diff
@@ -158,3 +190,60 @@ automation:
             - alert
             - badge
 ```
+
+### Notification color
+
+![android](assets/android.svg)
+In Android you can set the `color` of the notification.  You can use either the color name or the hex code
```
Comment (klaasnicolaas) [inline docs/notifications/basic.md]:
```suggestion
In Android you can set the `color` of the notification, you can use either the color name or the hex code.
```

**Comment by TomBrien** [comment]
@dshokouhi I just ported over some of the other changes so you'll just need to update this branch (should be a button below this). If you want to go with the review changes above just accept them. I'm happy to go ahead without as they are both style points (although I would like to remove the double space after a full stop).

**Comment by dshokouhi** [comment]
@TomBrien ok updated the branch and committed the suggestions.  I'll get the hang of our styling soon enough :)  I am a bit hesitant to merge this PR as I am not sure what is needed for the versioning.  Do I just copy over the changes to https://github.com/home-assistant/companion.home-assistant/tree/master/website/versioned_docs/version-2.0.0 Trying to follow your other PR's but not sure if I am missing something lol

**Comment by TomBrien** [comment]
@dshokouhi you’ve got it. Essentially just copy the content to the same place file under `version-2.0.0` but don’t change the meta data in the versioned file. I’ll take a look over this now.

**Comment by TomBrien** [comment]
I should also say I will probably take plunge and upgrade the site tomorrow so if you don’t want to worry about the versioned docs that’s fine. I’ll used the files in `docs/` when I migrate them

**Comment by dshokouhi** [comment]
@TomBrien i think that would be best :) I can hold off on PRs until the upgrade is complete :)

**Comment by TomBrien** [comment]
Ok well merge this when you’re happy and then I’ll do the upgrade in the morning

**Pull request merged**